config/routes.rb

```diff
@@ -2,6 +2,9 @@
   resources :acciones
   resources :accion_en_snapshots
   resources :snapshots
+  get 'importar_snapshot/establecer_origen' => 'snapshot_importer#establecer_origen', as: 'establecer_origen'
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.